### PR TITLE
the CSS2DObject visible is false but still show

### PR DESCRIPTION
**Pull request by callmegaga**
fixed when the CSS2DObject visible is false,But it still is shown.

### CODE REVIEW

File: examples/js/renderers/CSS2DRenderer.js [hunk 1]
```diff
@@ -82,7 +82,12 @@ THREE.CSS2DRenderer = function () {
 			element.style.MozTransform = style;
 			element.style.oTransform = style;
 			element.style.transform = style;
-			element.style.display = ( vector.z < - 1 || vector.z > 1 ) ? 'none' : '';
+			
+      if (!object.visible || vector.z < - 1 || vector.z > 1){
+      	element.style.display = 'none'
+      }else {
+      	element.style.display = ''
+      }
```
Comment (WestLangley) [inline examples/js/renderers/CSS2DRenderer.js]: How about
```js
element.style.display = ( ! object.visible || vector.z < - 1 || vector.z > 1 ) ? 'none' : '';
```

Comment (wwwYing) [reply examples/js/renderers/CSS2DRenderer.js]: YES,That's better.

**Comment by mrdoob** [comment]
I propose this instead 😇

```js
element.style.display = ( object.visible && vector.z > - 1 && vector.z < 1 ) ? '' : 'none';
```

**Comment by WestLangley** [comment]
Then you probably should use `<= 1` and `>= - 1`.

**Comment by mrdoob** [comment]
Will merge and amend.

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**